docs/translations.rst

```diff
@@ -418,16 +418,20 @@ In order to review the demo server as a *journalist*:
 How to become a reviewer
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can ask to become a reviewer for a language by posting a message
-in the `translations category of the SecureDrop forum`_.
+`Contact Localization Lab`_ to ask to join a `language team`_ as a reviewer.
 
 .. _how_to_add_a_new_language:
 
 How to add a new language to SecureDrop
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We love seeing SecureDrop translated into new languages. Just ask us
-to add yours by posting in the `translations category of the SecureDrop forum`_.
+We love seeing SecureDrop translated into new languages.  `Contact Localization
+Lab`_ to ask to start a new `language team`_ and have the new language added to
```

Comment:
Should this be "start a new language team or join an existing one"?

Reply:
Good catch; thanks.  I've done more or less the same thing (changed the order) in ffb7f21.